lib/gcs_signer.ex

```diff
@@ -23,27 +25,32 @@ defmodule GcsSigner do
 
   """
   @spec sign_url(
-    %{client_email: String.t, private_key: String.t},
-    String.t,
-    String.t,
-    sign_url_opts
-  ) :: String.t
+          %{client_email: String.t(), private_key: String.t()},
+          String.t(),
+          String.t(),
+          sign_url_opts
+        ) :: String.t()
   def sign_url(client, bucket, key, opts \\ []) do
     verb = opts[:verb] || "GET"
     md5_digest = opts[:md5_digest] || ""
     content_type = opts[:content_type] || ""
     expires = opts[:expires] || hours_after(1)
     resource = "/#{bucket}/#{key}"
 
-    signature = [verb, md5_digest, content_type, expires, resource]
-                |> Enum.join("\n") |> generate_signature(client)
+    signature =
+      [verb, md5_digest, content_type, expires, resource]
+      |> Enum.join("\n")
+      |> generate_signature(client)
 
     url = "#{@base_url}#{resource}"
-    qs = %{
-      "GoogleAccessId" => client.client_email,
-      "Expires" => expires,
-      "Signature" => signature
-    } |> URI.encode_query
+
+    qs =
```

Comment:
Separating righthand looks nice on the [signature](https://github.com/shakrmedia/gcs-signer-elixir/pull/4/files#diff-0ace863873912b6324af45646ac213a0R40), so thank you for adjusting it. 

On here however original code formatting feels better in terms of indentation. What do you think?